shaders/src/light_indirect.fs

```diff
@@ -13,12 +13,31 @@
 
 #define IBL_INTEGRATION_IMPORTANCE_SAMPLING_COUNT   64
 
+//------------------------------------------------------------------------------
+// IBL tinting helpers
+//------------------------------------------------------------------------------
+
+float OverlayBlend(float a, float b) {
+    // Visually almost-luminance-preserving formulation. This uses an artist-provided constant below
+    // that they've found the visually most pleasing under _most_ (but not all) circumstances.
+    const float kT = 0.65;
+    const float kOneOverT = 1.0 / kT;
+    return (a < kT) ? a * kOneOverT * b : a - kT + b;
+}
+vec3 OverlayBlend(vec3 a, vec3 b) {
+    return vec3(OverlayBlend(a.x, b.x), OverlayBlend(a.y, b.y), OverlayBlend(a.z, b.z));
+}
+
+vec3 TintIbl(vec3 color) {
+    return mix(color, OverlayBlend(color, frameUniforms.iblTintAndIntensity.rgb), frameUniforms.iblTintAndIntensity.w);
```

Comment:
@pixelflinger Not fond of adding that cost unconditionally on mobile. It's not huge but it's not nothing either. Wdyt?

Reply:
Maybe we should make this a material option? I think that would make sense because it's going to be used to rarely.

Reply:
WDYT about adding a material attribute that effectively corresponds to `kT` of `overlayBlend` (the highest intensity that we still overlay blend with the tint color)? I'd `#define` it and if that's zero I'm not tinting anything, otherwise, I'm tinting and now kT becomes a user adjustable parameter.